packages/SwingSet/src/kernel/state/kernelKeeper.js

```diff
@@ -779,6 +777,28 @@ export default function makeKernelKeeper(kvStore, streamStore, kernelSlog) {
     return vk;
   }
 
+  function vatIsAlive(vatID) {
+    insistVatID(vatID);
+    return kvStore.has(`${vatID}.o.nextID`);
+  }
+
+  /**
+   * Cease writing to the vat's transcript.
+   *
+   * @param { string } vatID
+   */
+  function closeVatTranscript(vatID) {
+    insistVatID(vatID);
+    const transcriptStream = `transcript-${vatID}`;
+    streamStore.closeStream(transcriptStream);
+  }
+
+  function evictVatKeeper(vatID) {
+    insistVatID(vatID);
+    closeVatTranscript(vatID);
```

Comment:
`vatWarehouse.evict()` is calling both `closeVatTranscript()` and `evictVatKeeper()`, so I think either it should just call `evictVatKeeper`, or this call to `closeVatTranscript` could go away. Not a big deal to do both (`streamStore.closeStream` should ignore the second one), but slightly redundant.